### PR TITLE
issue #2419 Ensure all python * imports pull from __all__ exports

### PR DESCRIPTION
**Pull request by aciffone23**

### CODE REVIEW

File: python/perspective/perspective/handlers/__init__.py [hunk 1]
```diff
@@ -11,18 +11,8 @@
 #  ┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛
 
 from .common import PerspectiveHandlerBase
+from .aiohttp import PerspectiveAIOHTTPHandler
```
Comment (timkpaine) [inline python/perspective/perspective/handlers/__init__.py:14]: These should remain in try/excepts.

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: python/perspective/perspective/handlers/__init__.py [hunk 2]
```diff
@@ -11,8 +11,20 @@
...
+    from .starlette import PerspectiveStarletteHandler
+except ImportError:
+    ...
+
+try:
+    from .tornado import PerspectiveTornadoHandler
+except ImportError:
+    ...
+
+__all__ = ["PerspectiveHandlerBase", "PerspectiveAIOHTTPHandler", "PerspectiveStarletteHandler", "PerspectiveTornadoHandler" ]
```
Comment (timkpaine) [inline python/perspective/perspective/handlers/__init__.py:30]: if the above imports fail, this will cause attribute errors when importing.

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: python/perspective/perspective/handlers/__init__.py (hunk 2 above)
Comment (aciffone23) [reply python/perspective/perspective/handlers/__init__.py:30]: Ok I was thinking about something like this, should deal with that, this way if it only puts it to __all__ if it passes. What do you think?

try:
    from .aiohttp import PerspectiveAIOHTTPHandler
    __all__.append("PerspectiveAIOHTTPHandler")
except ImportError:

**Comment by texodus** [comment]
Closing due to bitrot. Please feel free to reopen after the CLA process + review fixes.

**Pull request closed without merging**